frigate/camera/state.py

```diff
@@ -53,8 +53,19 @@ def __init__(
         self.callbacks = defaultdict(list)
         self.ptz_autotracker_thread = ptz_autotracker_thread
         self.prev_enabled = self.camera_config.enabled
+        self.requires_face_detection = (
+            self.config.face_recognition.enabled
+            and "face" not in self.config.objects.all_objects
```

Comment:
I recently changed this in lpr to be camera specific and using `self.config.cameras[camera].objects.track` so that users could be more specific about when they wanted lpr to run and with what model. Would that make sense to do here too?

Reply:
personally I don't think so, if you have a model that detects faces then you should be using the model with built in face detection as it will perform much better